### PR TITLE
Dispatcher: correctly set document.configurable

### PR DESCRIPTION
**Pull request by yparitcher**
This allows the ConfigDialog to update properly

@poire-z 
Please see if this helps :)

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7116)

### CODE REVIEW

**Comment by poire-z** [comment]
Looks like what I had in mind.
But I'm not a heavy dispatcher user to judge how it works.
Just did a quick test, associating to a gesture multiple actions:
- toggle nightmode (nothing to update)
- decreased font size: worked and bottom updated
- set line spacing to 80%: seems to apply to book, but bottom not updated (even after a quit/start)
- set kerning mode to off: it is applied on book, but bottom not updated (even after a quit/start)
(seems to still not work if I only set it alone to this gesture)

Unrelated:
- Line spacing number picker: long-press only increase by 2 (so, not much help vs tap which increase by 1 :)
- Seems I can't disable "Decrease font size by" ? :) I can just set the increment to 0, but it stays checked.

**Comment by poire-z** [comment]
Mhhh, it happened I tested this PR ... without actually applying it on my tree :) 
My above test now seems to all work fine :) Line spacing and kerning are properly reflected in the bottom menu :)
(May be the decrease font size worked because of the onSetFontSize existing handled I mentionned in the other PR?)

File: frontend/dispatcher.lua [hunk 1]
```diff
@@ -289,6 +290,9 @@ function Dispatcher:init()
         for y=1,#base[i].options do
             local option = base[i].options[y]
             if settingsList[option.name] ~= nil then
+                if option.name ~= nil and option.values ~= nil then
+                    settingsList[option.name].configurable = {name = option.name, values = option.values}
```
Comment (poire-z) [inline frontend/dispatcher.lua:294]: I haven't looked at all the actions that reflects in the bottom menu - but are there some that would not be handled? Or it just magically works for all of them?

**Review by poire-z — APPROVED** [review]

**Comment by yparitcher** [comment]
>     * Seems I can't disable "Decrease font size by" ? :) I can just set the increment to 0, but it stays checked.

hold to clear.

File: frontend/dispatcher.lua (hunk 1 above)
Comment (yparitcher) [reply frontend/dispatcher.lua:294]: the action from the bottom menu are:
https://github.com/koreader/koreader/blob/e157395660108032ca02c2acb3275d6012af23e7/frontend/dispatcher.lua#L134-L155

and they all are reflected here. ConfigDialog/Creopts use the same name value as configurable, and the values key it wat to set, (as opposed to args)

**Comment by poire-z** [comment]
> Line spacing number picker: long-press only increase by 2 (so, not much help vs tap which increase by 1 :)

@yparitcher : any reason you can think of to not increase it? (like .max being most often < 5?):
```diff
--- a/frontend/dispatcher.lua
+++ b/frontend/dispatcher.lua
@@ -418,3 +418,3 @@ function Dispatcher:addItem(caller, menu, location, settings, section)
                             value_step = 1,
-                            value_hold_step = 2,
+                            value_hold_step = 5,
                             value_max = settingsList[k].max,
@@ -460,3 +460,3 @@ function Dispatcher:addItem(caller, menu, location, settings, section)
                             value_step = 1,
-                            value_hold_step = 2,
+                            value_hold_step = 5,
                             value_max = settingsList[k].max,
```

**Comment by yparitcher** [comment]
I probabaly copied it from somewhere, so go ahead.

**Pull request merged**